persistent_stream.go

```diff
@@ -38,6 +38,12 @@ func (d *Daemon) handlePersistentConn(r ggio.Reader, unsafeW ggio.WriteCloser) {
 	d.terminateOnce.Do(func() { go d.awaitTermination() })
 
 	w := utils.NewSafeWriter(unsafeW)
+
+	if err := w.WriteMsg(&pb.Response{Type: pb.Response_OK.Enum()}); err != nil {
```

Comment:
@justheuristic Can you provide an example of a situation when sending OK here is useful?